frontend/lib/map/map/map.dart

```diff
@@ -56,6 +56,20 @@ class _MapState extends State<Map> {
 
   _onMapCreated(controller) => this._controller = controller;
 
+  Symbol _symbol;
+
+  _removeSymbol() async {
```

Comment:
There is probably a ?better? way of doing this instead of adding and removing the element. We could just add a property on the element on the map called `iconSize`. Our map style could then use this if it is set.

We can use your solution though if it does not make problems. Does the symbol disappear when scrolling away?

Reply:
>There is probably a ?better? way of doing this instead of adding and removing the element. We could just add a property on the element on the map called iconSize. Our map style could then use this if it is set.

Yeah, I don't like the way I draw a symbol on top of the original symbol either, but after a deep dive into the mapbox flutter library, I don't see any other possiblity (but if you do, I'd appreciate to know :) ). Afaics, there's no possibility to "change" the feature queried through the flutter bridge.

> We can use your solution though if it does not make problems. Does the symbol disappear when scrolling away?

Besides some very small spoardeous flickerings, it behaves as I would expect.

Reply:
Huch oke there is no way to write to the `"properties"` object within the queried features?

If that is the case then you are right. Really shitty lib...

Reply:
@elKei24 do you have experience with this?

Reply:
No, sadly not

Reply:
Is `featureInfo["properties"]` writeable? We would need to write into this map.

Reply:
well, it's writable, but it comes through a json string from the flutter bridge and hence there's no effect if you'd write anything on it. (I haven't tried it though :thinking: )

Reply:
I'm looking thourhg the API right now. Maybe updateSymbol would be nice: https://github.com/tobrun/flutter-mapbox-gl/blob/8f9838ff5db4e6d4c17e09a5088afb38430b693b/mapbox_gl_web/lib/src/mapbox_map_controller.dart#L170

Reply:
There is no way to write to the properties as far as i can see :(